src/plugins/wb-disable/disable.js

```diff
@@ -53,11 +67,19 @@ var componentName = "wb-disable",
 						localStorage.setItem( "wbdisable", "true" );
 					} catch ( e ) {}
 
-					// Append the Standard version link
-					li.innerHTML = "<a class='wb-sl' href='" + nQuery + "wbdisable=false'>" + i18n( "wb-enable" ) + "</a>";
+					// Temporary fix for i18n until text are translated
+					if ( !noticeHeader && wb.lang === "fr" ) {
+						noticeHeader = i18nTemporary.fr[ "disable-notice-h" ];
+						noticeBody = i18nTemporary.fr[ "disable-notice" ];
+					} else if ( !noticeHeader ) {
+						noticeHeader = i18nTemporary.en[ "disable-notice-h" ];
+						noticeBody = i18nTemporary.en[ "disable-notice" ];
+						noticehtml = noticehtml + " lang='en'";
+					}
```

Comment:
@lucas-hay considering the wb-disable is a real core feature of WET, I do think we should not introduce an aware WCAG issue especially when we have a work around for it. 
As it was mentioned in this PR description, that block of code is just a temporary fix until the text get inserted in the i18n file. 
By doing it, we would make fail WCAG all page that are not in English or French.

May be later, we can discuss about a better fallback for text that has been not translated yet.